### ISSUE TITLE
Support XSD

### ISSUE THREAD

**Issue by iaejean** [schmittjoh/serializer#602]
Hi, 

This is a great component very helpful, it could be great if it were posible generate a XSD for XML serialization or JSON-Schema for JSON serialization, the idea is provide a definition when the xml or json output is consumed by a third.

**Comment by goetas** [comment]
Check  https://github.com/goetas/xsd2php

**Comment by goetas** [comment]
Ah, you want to generate the schema from the metadata? If yes, then probably it will not be helpful

**Comment by iaejean** [comment]
Yes, I have seen xsd2php project, but, thinking in a REST API where I deserialized from XML o JSON, so, if I could provide the definition for XML or JSON  (XSD, JSON-Schema) to the consumers according to the Objects Definition (php2xsd generation in this case), I don't know if I'm clear, it's just an idea.

Sorry my bad English.
